### PR TITLE
Fixed up CHANGELOG

### PR DESCRIPTION
**Pull request by Aerijo**
Mostly just cosmetic. Started with a simple typo, then went through and fixed all the links. Internal links to issues/pulls are now only shown by `#<num>`. External links are now `<user>/<repo>#<num>` (this makes it obvious that some links take you to yudai-nkt's repo). I left people references alone.

### CODE REVIEW

**Comment by Aerijo** [comment]
Hmm. Could have sworn I made this into just a pull request, but it seems to have committed. Sorry about that, I was going to wait for someone else to review it first.

The changes are just cosmetic though, so there are no conflicts or end user differences.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-## 1.1.0
+## 1.1.0
```
Comment (yudai-nkt) [inline CHANGELOG.md]: This trailing whitespace should be removed.

**Comment by Aerijo** [comment]
Yeah, I did the changes in the raw view, and checked the result on the preview view. This didn't show differences, so I only noticed the whitespace after submitting the pull request and seeing the diff view. I wanted to fix it quickly, so I clicked the browser back button and made the change. I clicked the pull request button again, but must have misread it because it merged immediately. It seems the change was recognised, as the one I accidentally merged does not have the trailing whitespace.

Anyway, fixed this one. It should be mergable now, for all the difference it would make.

File: CHANGELOG.md (hunk 1 above)
Comment (Aerijo) [inline CHANGELOG.md]: Fixed

**Comment by yudai-nkt** [comment]
No need to be quick that much, just be careful. ;-)

**Pull request merged**